go/pkg/client/bytestream.go

```diff
@@ -42,6 +42,10 @@ func (c *Client) WriteChunked(ctx context.Context, name string, ch *chunker.Chun
 				return err
 			}
 			if chunk.Offset == 0 {
+				// Notice that the digest in the chunker might be misleading.
+				// Specifically, for compressed blob uploads, the resource
+				// name should include the uncompressed digest - while chunker
+				// should be including the compressed digest.
```

Comment:
I thought that you removed the Digest() function from Chunker altogether, instead relying on UE.digest

Reply:
Yes - this is just an artifact of rebasing :)